frontend/app/views/joiner/form/paymentPayPal.scala.html

```diff
@@ -14,56 +13,62 @@
     countriesWithCurrencies: List[CountryWithCurrency],
     idUser: IdentityUser,
     pageInfo: PageInfo,
-    trackingPromoCode: Option[PromoCode],
-    promoCodeToDisplay: Option[PromoCode],
     countryGroup: Option[CountryGroup],
     paypalTestVariant: String
     )(implicit token: play.filters.csrf.CSRF.Token, request: actions.AuthRequest[_])
 
-@main(plans.tier.cta, pageInfo=pageInfo, simpleHeader=true) {
+@main(plans.tier.cta, pageInfo=pageInfo, simpleHeader=true, margins=false) {
 
-    <main class="page-content skin-checkout-b">
+    <main class="page-content skin-checkout-b skin-checkout-paypal">
 
         @fragments.analytics.appNexusPixel("add", AppnexusPixel.checkoutPageids(plans.tier))
 
+        <!-- FORM TITLE -->
+        <section class="checkout-header">
+            <h1 class="checkout-headline">@fragments.form.formCta(plans.tier)</h1>
+        </section>
+
+        <!-- CHECKOUT FORM -->
         <form action="@routes.Joiner.joinPaid(plans.tier)" method="POST" id="payment-form" class="js-form" novalidate>
             @CSRF.formField
             <input type="hidden" name="tier" value="@plans.tier.slug"/>
 
-            <section class="form-header">
-                <h1 class="form-headline">@plans.tier.cta</h1>
-            </section>
-
+            <section class="form-content">
 
-            <section class="form-section form-section--no-padding">
-                <div class="form-section__content">
-                @fragments.form.errorMessageDisplay()
-                </div>
-            </section>
+                <!-- PARTNER/PATRON BENEFITS LIST -->
+                @if(plans.tier != Supporter()){
```

Comment:
I think `plans`, a.k.a. `MonthYearPlans`, can only be either `Supporter`, `Partner` or `Patron`.